docs/section21/Rule21-10.md

```diff
@@ -0,0 +1,69 @@
+
+# Boiler - Rule 21-10  
+
+**Rule ID:** 21-10  
+**Rule Description:** When the building uses boilers the hot water pump shall be modeled as riding the pump curve if the hot water system serves less than 120,000 ft^2 otherwise it shall be modeled with a VFD.  
+**Rule Assertion:** B-RMR  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.3.5 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12
+2. B-RMR is not modeled with purchased heating.
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11, or 12: `if PLACEHOLDER_AIRSIDE_SYSTEM_RULE-X-XX:`
+2. B-RMR is not modeled with purchased heating: `if NOT Rule-21-1:`
+
+## Rule Logic:  
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - Check if zone has HVAC terminals, get zone terminals: `if zone_b.terminals: zone_terminals_b = zone_b.terminals`
+
```

Comment:
We have several rules that involve determining the floor area served by the system (e.g., 21-5 and 21-10 for boilers). I recommend having a function to determine floor area served by a given HW/CHW plant.

Reply:
Agreed, we should have a function for that.

Reply:
I put a placeholder function in this RDS. It'll be better to wait for the team to finalize the schema about terminal connection before starting the function.